README.md

```diff
@@ -158,9 +166,599 @@ search.start();
+- ✅ showPrevious: Whether to display the previous page link.
+- ✅ showNext: Whether to display the next page link.
+- ✅ showLast: Whether to display the last page link.
+- ✅ padding: The number of pages to display on each side of the current page.
+- ✅ totalPages: The maximum number of pages to browse.
+- ✅ scrollTo: Where to scroll after a click. Set to false to disable.
+- ✅ templates: The templates to use for the widget.
+- ✅ cssClasses: The CSS classes to override.
+
+```js
+instantsearch.widgets.pagination({
+  container: '#pagination',
+})
+```
+
+### ✅ HitsPerPage
+
+[Hits per page references](https://www.algolia.com/doc/api-reference/widgets/hits-per-page/js/)
+
+The `hitsPerPage` widget displays a dropdown menu to let the user change the number of displayed hits.
+
+- ✅ container: The CSS Selector or HTMLElement to insert the widget into. _required_
+- ✅ [items](https://www.algolia.com/doc/api-reference/widgets/hits-per-page/js/#widget-param-items): The list of available options _required_
+- ✅ cssClasses: The CSS classes to override.
+- ✅ transformItems: function receiving the items, called before displaying them.
+
+### ❌ Breadcrumb
+
+[Breadcrumb references](https://www.algolia.com/doc/api-reference/widgets/breadcrumb/js/)
+
+The `breadcrumb` widget is a secondary navigation scheme that lets the user see where the current page is in relation to the facet’s hierarchy.
+
+No compatibility because MeiliSearch does not support hierarchical facets.
+
+If you'd like get nested facets implemented, please vote for it in the [roadmap](https://roadmap.meilisearch.com/c/97-nested-facets?utm_medium=social&utm_source=portal_share).
+
+### ✅ Stats
+
+[Stats references](https://www.algolia.com/doc/api-reference/widgets/stats/js/)
+
+The `stats` widget displays the total number of matching hits and the time it took to get them (time spent in the Algolia server).
+
+- ✅ container: The CSS Selector or HTMLElement to insert the widget into. _required_
+- ✅ cssClasses: The CSS classes to override.
+- ✅ transformItems: function receiving the items, called before displaying them.
+
+```js
+instantsearch.widgets.stats({
+  container: '#stats',
+})
+```
+
+### ❌ Analytics
+
+[Analytics](https://www.algolia.com/doc/api-reference/widgets/analytics/js/)
+
+Deprecated. See [Insight](#Insight).
+
+### ❌ QueryRuleCustomData
+
+[QueryRuleCustomData references](https://www.algolia.com/doc/api-reference/widgets/query-rule-custom-data/js/)
+
+You may want to use this widget to display banners or recommendations returned by Rules, and that match search parameters.
+
+No compatibility because MeiliSearch does not support [rules](https://www.algolia.com/doc/guides/managing-results/rules/rules-overview/).
+
+### ❌ QueryRuleContext
+
+[Query rule context references](https://www.algolia.com/doc/api-reference/widgets/query-rule-context/js/)
+
+The queryRuleContext widget lets you apply ruleContexts based on filters to trigger context-dependent Rules.
```

Comment:
```suggestion
The queryRuleContext widget lets you apply ruleContexts based on filters to trigger context-dependent rules.
```

Reply:
[Rules](https://www.algolia.com/doc/guides/managing-results/rules/detecting-intent/) are a feature of Algolia, should we not keep the capital letter in such cases?